File: front-end/src/app/domain/Team.ts
export class Team {
  constructor(public teamId: number,
              public abbrev: string,
              public nickname: string,
              public yearFounded: number,
              public city: string,
              public arena: string,
              public owner: string,
              public division: string,
              public icon: string
  ) {
  }
}

File: src/main/java/com/nba/datacenter/domain/Team.java
package com.nba.datacenter.domain;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Team {
    private int teamId;
    private String Abbrev;
    private String nickname;
    private int yearFounded;
    private String city;
    private String arena;
    private String owner;
    private String division;
    private String icon;
}
